pom.xml

```diff
@@ -1,42 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
-    <groupId>com.aperto.magkit</groupId>
+    <parent>
+        <groupId>com.ibmix.magkit</groupId>
+        <artifactId>magkit-pom</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
     <artifactId>magnolia-vanity-url</artifactId>
     <version>1.6.1-SNAPSHOT</version>
     <packaging>jar</packaging>
     <name>magnolia-vanity-url Magnolia Module</name>
 
     <inceptionYear>2013</inceptionYear>
 
-    <organization>
-        <name>Aperto – An IBM Company</name>
-        <url>https://www.aperto.com</url>
-    </organization>
-
     <properties>
-        <magnoliaVersion>6.2.12</magnoliaVersion>
-        <magnolia.content-types.version>1.2</magnolia.content-types.version>
-        <magnolia.site.version>1.4</magnolia.site.version>
         <qrgen.version>1.4</qrgen.version>
-
-        <source.javaversion>11</source.javaversion>
-        <target.javaversion>11</target.javaversion>
-
-        <source.encoding>UTF-8</source.encoding>
-        <project.build.sourceEncoding>${source.encoding}</project.build.sourceEncoding>
-
-        <!-- checkstyle can be disabled with this property -->
-        <checkstyle.skip>false</checkstyle.skip>
-        <checkstyle.config>src/config</checkstyle.config>
+        <checkstyle.config>com/de/ibmix</checkstyle.config>
```

Comment:
do we need this property here, or can it be set in magkit-pom?

Reply:
removed